terraform/ecs/variables.tf

```diff
@@ -39,6 +39,13 @@ variable "ecs_taskdef_network_mode" {
   default = "awsvpc"
 }
 
+# if patch is true, we will wait until the instance gets patched after launching the instance,
+# also set a patch tag onto the instance so that the instance get picked by the ssm patching process.
+# and then start the installation of collector.
+variable "patch" {
+  default = true
+}
```

Comment:
See comment further above. Patching is not done through tags. We may have to jump through more hoops to do the same type of patching on ecs ec2 instances. I don't consider this a blocker for this PR because it was not being done previously.

Reply:
Yes. I would remove the tag because it is not used in the same context as EC2.